boards/Parrot/Parrot.config

```diff
@@ -0,0 +1,53 @@
+# Configuration for a Parrot running Qubes and other OSes
+#
```

Comment:
A note here on how you obtained and reproduced the shrinked ifd would be helpful for people wanting to reproduce.

Reply:
@gompa would still be required!